File: yt/yt/flow/examples/java/url_downloader/url_downloader/src/main/java/tech/ytsaurus/flow/examples/urldownloader/model/ProcessedUrl.java
package tech.ytsaurus.flow.examples.urldownloader.model;

import javax.persistence.Entity;

// Field order must match the "processed_urls" stream definition in the static spec.
@Entity
public class ProcessedUrl {
    private String host;
    private String url;
    private String data;

    public ProcessedUrl() {
    }

    public ProcessedUrl(String host, String url, String data) {
        this.host = host;
        this.url = url;
        this.data = data;
    }

    public String getHost() {
        return host;
    }

    public void setHost(String host) {
        this.host = host;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }
}

File: yt/yt/flow/examples/kotlin/url_downloader/url_downloader/src/main/kotlin/tech/ytsaurus/flow/examples/urldownloader/model/ProcessedUrl.kt
package tech.ytsaurus.flow.examples.urldownloader.model

import javax.persistence.Entity

// Field order must match the "processed_urls" stream definition in the static spec.
@Entity
class ProcessedUrl {
    var host: String? = null

    var url: String? = null

    var data: String? = null

    constructor()

    constructor(host: String?, url: String?, data: String?) {
        this.host = host
        this.url = url
        this.data = data
    }
}
